méli-mélo/th-vitality/vitality.html

```diff
@@ -0,0 +1,189 @@
+---
+title: Vitality theme
+dateModified: 2025-01-17
+description: Vitality CSS styles
+lang: en
+pageclass: cnt-wdth-lmtd
+css:
+- style.css
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.css
+script:
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.js
+layout: without-h1
+---
+<h1 property="name" id="wb-cont" class="vitality">{{ page.title }}</h1>
+<p>The colors offered by this theme aim to evoke a sense of vitality for the user during certain life events, such as the addition of a child to your family.</p>
+{% assign metadata = site.pages | where: "output", "false" | where: "componentName", "th-vitality" | first %}
+<p>Sponsor: {{ metadata.sponsor }}</p>
+<p>Accessibility statement: {{ metadata.a11yStatement }}</p>
+<p>Progressive enhancement note:</p>
+<ul>
+    {% for peNote in metadata.peNote %}
+    <li>{{ peNote }}</li>
+    {% endfor %}
+</ul>
+
+<h2>CSS classes</h2>
+<dl>
+    <dt><code>.vitality</code></dt>
+    <dd>When applied to the <code>&lt;h1&gt;</code> element it sets a green underline to the title element of the page.</dd>
+    <dd>When applied to the Steps Quiz along with the <code>wb-steps</code> and <code>quiz</code> it set the thematic colors to the navigation buttons, the progress bar and to the <code>&lt;legend&gt;</code> text.</dd>
+    <dt><code>.cards</code></dt>
+    <dd>Cards can be used with <code>.light-hover</code> or <code>.dark-hover</code> to change the colours on hover or active states.</dd>
+	<dt><code>.text-vitality</code></dt>
+    <dd>Style the content with the primary thematic color.</dd>
+	<dt><code>.bg-vitality</code></dt>
+    <dd>Style the content's background with the primary thematic color.</dd>
+	<dt><code>.bg-vitality-light</code></dt>
+    <dd>Style the content's background with the secondary thematic color.</dd>
+	<dt><code>.btn-vitality</code></dt>
+    <dd>Along with <code>.btn-primary</code> or <code>.btn-default</code>, set a green colour thematic to buttons.</dd>
+	<dt><code>.printable-checklist</code></dt>
+    <dd>Displays a checkbox besides open summary elements on print media.</dd>
+</dl>
+
+<h2>Examples</h2>
+<h3>Green Underline to the <code>&lt;h1&gt;</code> </h3>
+<p>The example has been applied to the <code>&lt;h1&gt;</code> element of this page.</p>
+<h4>Code</h4>
+<pre><code>&lt;h1 property="name" id="wb-cont" class="vitality"&gt;&lt;-- Page title --&gt;&lt;/h1&gt;</code></pre>
+
+<h3>Cards</h3>
+<p>Example of cards used in a grid layout.</p>
+<div class="row">
```

Comment:
The examples for the cards are not working. They are missing a wrapping element with the class "vitality".
```suggestion
<div class="row vitality">
```

Reply:
Done! I've made the suggested change and added to the code section sample as well.